stuffer/s2n_stuffer_text.c

```diff
@@ -45,6 +45,7 @@ int s2n_stuffer_peek_check_for_str(struct s2n_stuffer *s2n_stuffer, const char *
 
 int s2n_stuffer_skip_whitespace(struct s2n_stuffer *s2n_stuffer)
```

Comment:
This function should be changed to `S2N_RESULT s2n_stuffer_skip_whitespace(struct s2n_stuffer *s2n_stuffer, uint32_t* chars_skipped)`

Reply:
Done!